README.adoc

```diff
@@ -15,26 +15,59 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
-* What's different from https://github.com/se-edu/addressbook-level3[level 3]:
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+== About
+
+This is a module planner application.
```

Comment:
Any reason why the list syntax (`- * blah blah`) was dropped? I think points can be presented clearer in list form.

Reply:
Ok reverted back to points. I didn't do the same for the "So tedious" and "Much better!" cos it seems kinda odd for them to be in points

Reply:
Current one is fine. Thanks!